cmd/adduser.go

```diff
@@ -108,6 +108,10 @@ type AddUserParams struct {
 	src           []string
 	tags          []string
 	credsFilePath string
+	userName      string
+	create        bool
+	keyPath       string
```

Comment:
It seems that create should be called createKey or be deleted as it seems that his is redundant with keyPath being "" or not. 
In the code below what would it mean if creat == false and keyPath == ""?
Add a function (p *AddUserParams) createKey() bool {return p.keyPath == ""}?